WorkloadManagementSystem/Service/SandboxStoreHandler.py

```diff
@@ -48,450 +49,452 @@ def initialize( self ):
     """
     if not entitySetup:
-      entitySetup = self.serviceInfoDict[ 'clientSetup' ]
+      entitySetup = self.serviceInfoDict['clientSetup']
     credDict = self.getRemoteCredentials()
-    return sandboxDB.assignSandboxesToEntities( enDict, credDict[ 'username' ], credDict[ 'group' ], entitySetup,
-                                                ownerName, ownerGroup )
+    return sandboxDB.assignSandboxesToEntities(enDict, credDict['username'], credDict['group'], entitySetup,
+                                               ownerName, ownerGroup)
 
   ##################
   # Unassign sbs to jobs
 
-  types_unassignEntities = [ ( list, tuple ) ]
-  def export_unassignEntities( self, entitiesList, entitiesSetup = False ):
+  types_unassignEntities = [(list, tuple)]
+
+  def export_unassignEntities(self, entitiesList, entitiesSetup=False):
     """
     Unassign a list of jobs
     """
     if not entitiesSetup:
-      entitiesSetup = self.serviceInfoDict[ 'clientSetup' ]
+      entitiesSetup = self.serviceInfoDict['clientSetup']
     credDict = self.getRemoteCredentials()
-    return sandboxDB.unassignEntities( { entitiesSetup : entitiesList }, credDict[ 'username' ], credDict[ 'group' ] )
+    return sandboxDB.unassignEntities({entitiesSetup: entitiesList}, credDict['username'], credDict['group'])
 
   ##################
   # Getting assigned sandboxes
 
-  types_getSandboxesAssignedToEntity = [ basestring ]
-  def export_getSandboxesAssignedToEntity( self, entityId, entitySetup = False ):
+  types_getSandboxesAssignedToEntity = [basestring]
+
+  def export_getSandboxesAssignedToEntity(self, entityId, entitySetup=False):
     """
     Get the sandboxes associated to a job and the association type
     """
     if not entitySetup:
-      entitySetup = self.serviceInfoDict[ 'clientSetup' ]
+      entitySetup = self.serviceInfoDict['clientSetup']
     credDict = self.getRemoteCredentials()
-    result = sandboxDB.getSandboxesAssignedToEntity( entityId, entitySetup,
-                                                     credDict[ 'username' ], credDict[ 'group' ] )
-    if not result[ 'OK' ]:
+    result = sandboxDB.getSandboxesAssignedToEntity(entityId, entitySetup,
+                                                    credDict['username'], credDict['group'])
+    if not result['OK']:
       return result
     sbDict = {}
-    for SEName, SEPFN, SBType in result[ 'Value' ]:
+    for SEName, SEPFN, SBType in result['Value']:
       if SBType not in sbDict:
-        sbDict[ SBType ] = []
-      sbDict[ SBType ].append( "SB:%s|%s" % ( SEName, SEPFN ) )
-    return S_OK( sbDict )
-
+        sbDict[SBType] = []
+      sbDict[SBType].append("SB:%s|%s" % (SEName, SEPFN))
+    return S_OK(sbDict)
 
   ##################
   # Disk space left management
 
   types_getFreeDiskSpace = [basestring]
-  def export_getFreeDiskSpace( self, path, size = 'TB' ):
+
+  def export_getFreeDiskSpace(self, path):
```

Comment:
I will do it because I am working against your branch but...
* The docstring needs updating
* In all the plugins, you wrote `MB` or `Should be MB`. Shall we take the opportunity and make it MB or TB everywhere ? 
If up to now srm was returning TB, we shall do TB, because probably many plots rely on that